### PR TITLE
added contributing file

### PR DESCRIPTION
**Pull request by Bridgetamana**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Contributing to ai-helper-tools
+
+Thank you for considering contributing to ai-helper-tools! We welcome contributions from everyone. Here’s a guide to help you get started.
```
Comment (Bashamega) [inline contributing.md:3]:
```suggestion
Thank you for considering contributing to Gemini tools! We welcome contributions from everyone. Here’s a guide to help you get started.
```

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+1. **Fork and Clone**:
+   ```bash
+   git clone https://github.com/Bashamega/ai-helper-tools.git
+   cd ai-helper-tools
+   ```
+2. **Install Dependencies**:
+   ```bash
+   npm install
+   ```
```
Comment (Bashamega) [inline contributing.md:62]: Add the step where the users setup the .env file

**Review by Bashamega — CHANGES_REQUESTED** [review]

File: contributing.md (hunk 2 above)
Comment (Bridgetamana) [reply contributing.md:62]: How? please

Comment (Bashamega) [reply contributing.md:62]: add the setups of copying the `.env.example` and with which key should they add and how to get it

**Review by Bashamega — APPROVED** [review]

**Pull request merged**